README.md

```diff
@@ -143,6 +143,23 @@ context:
     rpmOstreeTreefile: centos-bootc/centos-bootc.yaml
 ```
 
+The configuration file can specify a containerfile to extract a base image
```

Comment:
So this talks about two places, the `containerfile` in `context` and `varsFromContainerfile` in `contentOrigin`, right?